ansible/roles/openshift_sso_app_zabbix/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "item: openshift.master.cluster.synthetic.exists"
+  zbx_host:
+    zbx_server: "{{ ozcs_zbx_server }}"
+    zbx_user: "{{ ozcs_zbx_user }}"
+    zbx_password: "{{ ozcs_zbx_password }}"
+    state: list
+    hostname: "{{ oss_clusterid }}-synthetic"
+  register: out
+
+- name: "item: oso.sso.app.aggregate"
+  when: out.results.results | length == 1
+  zbx_item:
+    zbx_server: "{{ ozcs_zbx_server }}"
+    zbx_user: "{{ ozcs_zbx_user }}"
+    zbx_password: "{{ ozcs_zbx_password }}"
+    name: oso.sso.app.aggregate
+    key: 'grpsum[ops-aws,sso.monitoring.container.running,sum,15m]'
+    value_type: int
+    data_type: decimal
+    description: "The sum of oso-monitor-sso containers checking in."
+    multiplier: ''
+    hostname: "{{ item }}"
+    zabbix_type: aggregate
+    interval: 900
+    delta: 0
+    units: '%'
+  with_items: "{{ ozcs_masters }}"
+  delegate_to: "{{ oss_clusterid }}-synthetic"
+  run_once: true
+
+- name: "SSO containers missing information for more than 15 minutes"
+  expression: "{ops-aws-synthetic:grpsum[ops-aws,sso.monitoring.container.running,sum,15m].min(#3)}<0"
```

Comment:
I'm not sure how often zabbix aggregate item datapoints get added, but it might be more meaningful to have it be across a time frame instead of a number of samples. Also, we want the alert to happen if we don't have 3 containers. This check is supposed to detect when one or more of the required 3 isn't running. So maybe .max(1800) } < 3. That would say that if during the entire previous half hour, there were never 3 containers running, alert.